song_player/songplayer.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <QWidget>
+#include <QMediaPlayer>
+#include <QAbstractButton>
+#include <QBoxLayout>
+#include <QStandardPaths>
+#include <QFileDialog>
+#include <QFileInfo>
+#include <QShortcut>
+#include <QCoreApplication>
+#include <QToolButton>
+#include <QTime>
+#include <QSlider>
+#include <QLabel>
+#include <QMouseEvent>
+#include <QStyle>
+
+QT_FORWARD_DECLARE_CLASS(QLabel)
+QT_FORWARD_DECLARE_CLASS(QSlider)
```

Comment:
http://article.gmane.org/gmane.comp.lib.qt.general/13667 they should be replaced with normal namespaces.